src/main/java/JacksonModels/LabelNames.java

```diff
@@ -0,0 +1,219 @@
+package JacksonModels;
```

Comment:
before using the auto-generated POJOs, it's essential to understand the JSON structure returned by the API, this helps in knowing which classes are relevant and how they are interconnected